README.md

```diff
@@ -6,5 +6,14 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a framework for building websites. As such, Rails establishes conventions for easier collaboration and maintenance. These conventions are codified as the Rails API (the application programming interface, or directives that control the code). The Rails API is documented online and described in books, articles, and blog posts. Learning Rails means learning how to use the Rails conventions and its API.
```

Comment:
Sort of. Rails is a framework for building _applications_, not just websites. It gives you tools and templates to easily build back-end and full-stack apps by providing a structure for request handling, database manipulation, and response generation.